docs/src/explanations/connection-operators.md

```diff
@@ -0,0 +1,364 @@
+
+```scala mdoc:silent:reset
+import chisel3._
+import chisel3.stage.ChiselStage
+import chisel3.util.DecoupledIO
+
+class Wrapper extends Module{
+  val io = IO(new Bundle {
+  val in = Flipped(DecoupledIO(UInt(8.W)))
+  val out = DecoupledIO(UInt(8.W))
+  })
+  val p = Module(new PipelineStage)
+  val c = Module(new PipelineStage) 
+  //connect Producer to IO
+  io.in := DontCare
+  p.io.a <> DontCare
+  val tmp = Wire(Flipped(DecoupledIO(UInt(8.W))))
+  tmp := DontCare
+  p.io.a <> io.in
+  // connect producer to consumer
+  c.io.a <> p.io.b
+  //connect consumer to IO
+  io.out <> c.io.b
+}
+class PipelineStage extends Module{
+  val io = IO(new Bundle{
+    val a = Flipped(DecoupledIO(UInt(8.W)))
+    val b = DecoupledIO(UInt(8.W))
+  })
+  io.b <> io.a
+}
+```
+Below we can see the resulting error for this example:
+```scala mdoc
+ChiselStage.emitVerilog(new Wrapper)
+```
+### Conclusion:
+If `<>` were used to assign the unidrectioned wire `tmp` to DontCare, we would get an error. But in the example above, we used `:=` and no errors occurred.
+But when `:=` was used to assign the wire to DontCare, no errors will occur.
+
+Thus, when assigning `DontCare` to a `Wire`, always use `:=`.
+
+
+##  Concept 4: You can use `<>` or `:=` to assign `DontCare` to directioned things (IOs)
+When assigning `DontCare` to something that is directioned, should you use `:=` or `<>`? 
+We will find out using the sample codes below:
+( Scastie link for the experiment:https://scastie.scala-lang.org/Shorla/ZIGsWcylRqKJhZCkKWlSIA/1)
+
+```scala mdoc:silent:reset
+import chisel3._
+import chisel3.stage.ChiselStage
+import chisel3.util.DecoupledIO
+
+class Wrapper extends Module{
+  val io = IO(new Bundle {
+  val in = Flipped(DecoupledIO(UInt(8.W)))
+  val out = DecoupledIO(UInt(8.W))
+  })
+  val p = Module(new PipelineStage)
+  val c = Module(new PipelineStage) 
+  //connect Producer to IO
+  io.in := DontCare
+  p.io.a <> DontCare
+  val tmp = Wire(Flipped(DecoupledIO(UInt(8.W))))
+  tmp := DontCare
+  p.io.a <> io.in
+  // connect producer to consumer
+  c.io.a <> p.io.b
+  //connect consumer to IO
+  io.out <> c.io.b
+}
+class PipelineStage extends Module{
+  val io = IO(new Bundle{
+    val a = Flipped(DecoupledIO(UInt(8.W)))
+    val b = DecoupledIO(UInt(8.W))
+  })
+  io.b <> io.a
+}
+```
+Below we can see the resulting Verilog for this example:
+```scala mdoc
+ChiselStage.emitVerilog(new Wrapper)
+```
+### Conclusion: 
+Both `<>` and `:=` can be used to assign directioned things (IOs) to DontCare as shown in `io.in` and `p.io.a` respectively. This is basically equivalent because in this case both `<>` and `:=` will determine the direction from the LHS.
+
+
+## Concept 5: `<>`  works between things with at least one known flow (An IO or child's IO). 
+If there is at least one known flow what will <> do? This will be showed using the experiment code.
+( Scastie link for the experiment:https://scastie.scala-lang.org/Shorla/gKx9ReLVTTqDTk9vmw5ozg)
```

Comment:
```suggestion
## Concept 5: `<>`  works between things with at least one known flow (An IO or child's IO). 

If there is at least one known flow what will `<>` do? This will be shown using the experiment code below:
( Scastie link for the experiment:https://scastie.scala-lang.org/Shorla/gKx9ReLVTTqDTk9vmw5ozg)
```